src/Explorer/Tabs/CloudShellTab/ShellTypes/CassandraShellHandler.tsx

```diff
@@ -41,7 +41,7 @@ export class CassandraShellHandler extends AbstractShellHandler {
     return `cqlsh ${getHostFromUrl(this._endpoint)} 10350 -u ${dbName} -p ${this._key} --ssl`;
   }
 
-  public getTerminalSuppressedData(): string {
-    return "";
+  public getTerminalSuppressedData(): string[] {
+    return [""];
```

Comment:
why is it changed to `array`, if you are planning to supress multiple, things, please mention it in PR description.